debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -225,6 +225,16 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withValidation(Field::isOptional)
             .withDescription("The SQL Server instance name");
 
+    public static final Field DATABASE_NAME = RelationalDatabaseConnectorConfig.DATABASE_NAME
+            .withNoValidation();
+
+    public static final Field DATABASE_NAMES = Field.create(DATABASE_CONFIG_PREFIX + "names")
+            .withDisplayName("Databases")
+            .withType(Type.STRING)
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.HIGH)
+            .withDescription("The names of the databases the connector should be monitoring");
```

Comment:
```suggestion
            .withDescription("The names of the databases from which the connector should capture changes");
```

To avoid confusion, we want to use the term "monitoring" only in the observability context, e.g. when referring to monitoring the connectors via JMX. It's not applied consistently yet, but let's use the right terminology for changed/new code. That said, perhaps you can update the `dbname` description accordingly, too?

Reply:
There's still a bunch of comments and descriptions in the configuration class where "monitoring" is used. I'll make the changes where suggested but rewording the rest deserves a separate issue.